### PR TITLE
add missing   unit names

### PR DESCRIPTION
**Pull request by Mrlinkwii**
add missing Scavenger unit names

### CODE REVIEW

File: language/units_en.lua [hunk 1]
```diff
@@ -412,6 +412,11 @@ return {
 				corroy = "Enforcer",
 				corsb = "Dam Buster",
 				corscreamer = "Screamer",
+				corscavdrag="Scav Dragon's Teeth",
+				corscavfort="Scav fortification wall",
+				corscavdtm="Scav missile wall",
+				corscavdtl="Scav Dragon's claw",
+				corscavdtf="Scav fire teeth",
```
Comment (sprunk) [inline language/units_en.lua:419]: Scav Dragon's Maw?

Comment (Mrlinkwii) [reply language/units_en.lua]: fixed :+1:

File: language/units_en.lua [hunk 2]
```diff
@@ -412,6 +412,11 @@ return {
 				corroy = "Enforcer",
 				corsb = "Dam Buster",
 				corscreamer = "Screamer",
+				corscavdrag="Scav Dragon's Teeth",
+				corscavfort="Scav fortification wall",
+				corscavdtm="Scav missile wall",
+				corscavdtl="Scav Dragon's claw",
+				corscavdtf="Scav Dragon's Maw",
```
Comment (WatchTheFort) [inline language/units_en.lua:419]: "Scav" should be removed, as these are not technically Scavenger units - those would be `corscavdrag_scav`, `corscavfort_scav`, etc. Scavenger unit names are then handled separately, see line 11.

File: language/units_en.lua [hunk 3]
```diff
@@ -412,6 +412,11 @@ return {
 				corroy = "Enforcer",
 				corsb = "Dam Buster",
 				corscreamer = "Screamer",
+				corscavdrag="Scav Dragon's Teeth",
+				corscavfort="Scav fortification wall",
+				corscavdtm="Scav missile wall",
+				corscavdtl="Scav Dragon's claw",
```
Comment (WatchTheFort) [inline language/units_en.lua:418]: Capitalization should be consistent with other unit names

**Review by WatchTheFort — CHANGES_REQUESTED** [review]

**Review by WatchTheFort — APPROVED** [review]

**Pull request merged**